src/media/css/app-list.styl

```diff
@@ -13,7 +13,7 @@ $app-list-background-color = #CBCDCD;
     left: 0;
 
     width: 100%;
-    overflow: auto;
+    overflow: hidden;
```

Comment:
I don't like relying on JS to scroll but the scrollbar on TV shouldn't occupy space either so this makes sense.